### PR TITLE
Removes RPC Instances

### LINKED ISSUE #932: Remove all instances of RPC
With the birth of [Quasar](https://github.com/SCE-Development/Quasar), RPC code across Core-v4 shall cease to exist to onboard the new era of peripherals solutions.

We can remove all instances of it in files such as below.
2021-12-25 at 5 57 01 PM

### PR DESCRIPTION
**Pull request by allen-tran**
## Resolves #932

- [X] Removed RPC urls from config example
- [X] Removed entire axios functions that contained RPC related activities
- [x] Removed RPC imports across files

Question: Is it necessary to remove worker-rpc module from `package-lock.json`?

### CODE REVIEW

**Comment by neelp03** [comment]
I checked all files in core-v4 for mentions of RPC and seems like you have removed all of them 👍

**Review by neelp03 — APPROVED** [review]
I checked all files in core-v4 for mentions of RPC and seems like you have removed all of them 👍

File: src/APIFunctions/3DPrinting.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 import { ApiResponse } from './ApiResponses';
 let config = require('../config/config.json');
-let GENERAL_API_URL = process.env.NODE_ENV === 'production' ?
-  config.GENERAL_API_URL_PROD : config.GENERAL_API_URL;
-let RPC_API_URL = process.env.NODE_ENV === 'production' ?
-  config.RPC_API_URL_PROD : config.RPC_API_URL;
+let GENERAL_API_URL =
+  process.env.NODE_ENV === 'production'
+    ? config.GENERAL_API_URL_PROD
+    : config.GENERAL_API_URL;
```
Comment (allen-tran) [inline src/APIFunctions/3DPrinting.js:7]: it appears my autoformatter is correcting a bunch of stuff. i will look into this

File: api/config/config.example.json [hunk 2]
```diff
@@ -15,9 +15,7 @@
   "googleApiKeys": {
     "CLIENT_ID": "XXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
     "CLIENT_SECRET": "XXXXXXXXXXXXXXXXXXXX",
-    "REDIRECT_URIS": [
-      "https://developers.google.com/oauthplayground"
-    ],
+    "REDIRECT_URIS": ["https://developers.google.com/oauthplayground"],
```
Comment (evanugarte) [inline api/config/config.example.json:18]: you really should turn off ur auto formatter and just rely on cmd k cmd f

File: src/APIFunctions/2DPrinting.js [hunk 3]
```diff
@@ -99,7 +100,8 @@ export async function printPage(data) {
  */
 export async function logPrintRequest(data) {
   let status = new ApiResponse();
-  await axios.post(PERIPHERAL_API_URL + '/PrintLog/addPrintLog', data)
+  await axios
```
Comment (evanugarte) [inline src/APIFunctions/2DPrinting.js:103]: remove all these random changes

File: src/APIFunctions/3DPrinting.js (hunk 1 above)
Comment (evanugarte) [reply src/APIFunctions/3DPrinting.js:7]: turn it off turn it off

File: api/config/config.example.json [hunk 4]
```diff
@@ -47,6 +47,5 @@
   "secretKey": "super duper secret key",
   "GENERAL_API_URL_PROD": "http://mainendpoints:8080/api",
   "PERIPHERAL_API_URL_PROD": "http://sce-peripheral-api:8081/peripheralapi",
-  "MAILER_API_URL_PROD": "http://sce-cloud-api:8082/cloudapi",
-  "RPC_API_URL_PROD": "http://localhost:8083/SceRpcApi"
-}
+  "MAILER_API_URL_PROD": "http://sce-cloud-api:8082/cloudapi"
+}
```
Comment (evanugarte) [inline api/config/config.example.json:51]:
```suggestion
}

```

Comment (evanugarte) [reply api/config/config.example.json:51]: nvm just fix this in #934

File: src/config/config.example.json [hunk 5]
```diff
@@ -2,13 +2,10 @@
   "GENERAL_API_URL": "http://localhost:8080/api",
   "PERIPHERAL_API_URL": "http://localhost:8081/peripheralapi",
   "MAILER_API_URL": "http://localhost:8082/cloudapi",
-  "RPC_API_URL": "http://localhost:8083/SceRpcApi",
   "DISCORD_SJSU_API_URL": "http://localhost:8080/api",
   "GENERAL_API_URL_PROD": "https://sce.engr.sjsu.edu/api",
   "PERIPHERAL_API_URL_PROD": "https://sce.engr.sjsu.edu/peripheralapi",
   "MAILER_API_URL_PROD": "https://sce.engr.sjsu.edu/cloudapi",
-  "RPC_API_URL_PROD": "https://sce.engr.sjsu.edu/SceRpcApi",
-  "DISCORD_SJSU_API_URL_PROD": "http://XXXXXXXXXXXX.XXX:8080/api",
```
Comment (evanugarte) [inline src/config/config.example.json:11]: why was this removed

File: util/setup-scripts/setup.sh [hunk 6]
```diff
@@ -4,11 +4,6 @@
 cd api
 # install dependencies
 npm install
-# remove any existing printRPC directory
-rm -rf printingRPC/
-# clone printingRPC repository
-git clone https://github.com/jerrylee17/printingRPC.git
-cd printingRPC
 # run the repo's setup script
```
Comment (evanugarte) [inline util/setup-scripts/setup.sh:7]: this line and everything after is also related to rpc pls remove

File: util/setup-scripts/setup.bat [hunk 7]
```diff
@@ -4,11 +4,6 @@ REM cd into our backend folder
 cd api
 REM install dependencies
 call npm install
-REM remove any existing printingRPC directory
-del printingRPC /F /Q
-REM clone printingRPC repository
-git clone https://github.com/jerrylee17/printingRPC.git
-cd printingRPC
 REM run the repo's setup script
```
Comment (evanugarte) [inline util/setup-scripts/setup.bat:7]: this line and everything after is also related to rpc pls remove

File: src/APIFunctions/2DPrinting.js [hunk 8]
```diff
@@ -81,7 +76,8 @@ export function parseRange(pages, maxPages) {
  */
 export async function printPage(data) {
   let status = new ApiResponse();
-  await axios.post(PERIPHERAL_API_URL + '/Printer/sendPrintRequest', data)
+  await axios.post(PERIPHERAL_API_URL + '/Printer/sendPrintRequest',
```
Comment (evanugarte) [inline src/APIFunctions/2DPrinting.js:79]: if possible manually undo all these unrelated changes, we can do this in a different eslint fix pr

File: src/APIFunctions/3DPrinting.js [hunk 9]
```diff
@@ -49,33 +47,6 @@ export async function submit3DPrintRequest(printRequest) {
...
- * @param {Object} printRequest - The request containing all of the project's
- * information.
- * @param {string} printRequest.raw - The raw STL file of the model.
- * @param {string} printRequest.name - The user's full name.
- * @param {string} printRequest.volume - The volume of the print in cubic cm.
- * @param {string} printRequest.copies - How many copies the user wants.
- * @returns {ApiResponse} - Containing any error information
- *                          related to the request.
- */
-export async function print3DModel(printRequest) {
```
Comment (evanugarte) [inline src/APIFunctions/3DPrinting.js:63]: i so wish this worked

**Review by evanugarte — APPROVED** [review]
